AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -81,6 +81,8 @@ belaruspartisan.by##.news-detail a[href="https://t.me/belaruspartisanby"]
 mywebpc.ru#?#.sidebar_list > div.widget:has(> div.textwidget > a[href="tg://resolve?domain=mywebpc"])
 ! https://github.com/AdguardTeam/AdguardFilters/issues/55890
 sankei.com###snsFollow
+! https://github.com/AdguardTeam/AdguardFilters/issues/61973
+digitalcitizen.life##.block-next-sidebar-subscription-options > :not(.subscription-options__social_buttons)
```

Comment:
for subscriptions we have the section
https://github.com/AdguardTeam/AdguardFilters/blob/c4bbe243c52e19d72f7bdee07be6529658f225ea/AnnoyancesFilter/sections/annoyances.txt#L1476